### PR TITLE
Admins can no longer jump to people while in the lobby.

### PR DESCRIPTION
**Pull request by NonQueueingMatt**
Doesn't need a changelog, does it?

### CODE REVIEW

**Comment by NonQueueingMatt** [comment]
!review

**Comment by NonQueueingMatt** [comment]
!bugfix

File: code/modules/admin/verbs/adminjump.dm [hunk 1]
```diff
@@ -24,8 +28,13 @@
 /client/proc/jumptoturf(var/turf/T in turfs)
 	set name = "Jump to Turf"
 	set category = "Admin"
+
 	if(!check_rights(R_ADMIN|R_MOD|R_DEBUG|R_DEV))
 		return
+
+	if(istype(usr, /mob/abstract/new_player))
```
Comment (mikomyazaki) [inline code/modules/admin/verbs/adminjump.dm]:
```suggestion
	if(isnewplayer(usr))
```

Apparently we have a helper for this. :shrug:

File: code/modules/admin/verbs/adminjump.dm [hunk 2]
```diff
@@ -43,6 +52,9 @@
 	if(!check_rights(R_ADMIN|R_MOD|R_DEBUG|R_DEV))
 		return
 
+	if(istype(usr, /mob/abstract/new_player))
```
Comment (mikomyazaki) [inline code/modules/admin/verbs/adminjump.dm]:
```suggestion
	if(isnewplayer(usr))
```

**Comment by Pratepresidenten** [comment]
Finally, some good food.

**Pull request merged**